webpack/InsightsHostDetailsTab/NewHostDetailsTab.js

```diff
@@ -20,7 +20,6 @@ import {
   selectSearch,
   selectHits,
 } from '../InsightsCloudSync/Components/InsightsTable/InsightsTableSelectors';
-import './InsightsTab.scss';
```

Comment:
Should we also remove the scss file itself?

Reply:
it's still needed for the insights tab in the old host page, when it'll be gone we can remove these files.